libndt-client.cpp

```diff
@@ -100,38 +100,33 @@ nearby servers. The default policy is `geo-options`. The deprecated
 `-random` flag is an alias for `-lookup-policy random`.
 
 You MUST specify what subtest to enable. The `-download` flag enables the
-download subtest. The `-upload` flag enables the upload subtest. The
-`-download-ext` flag enables the multi-stream download subtest, which
-is not implemented by M-Lab servers. Hence you need to know the hostname
-of a server implementing this feature to run the test with.
+download subtest. The `-upload` flag enables the upload subtest.
 
 The `-port <port>` flag specifies what port to use. The default is to
 use the correct port depending on the selected NDT protocol (see below).
 
-By default, we use the most ancient NDT protocol. However, adding the
-`-json` flag enables wrapping NDT messages inside of JSON objects. Adding
-the `-tls` flag causes NDT to use a TLS connection rather than a TCP
-connection. When using `-tls`, you may also want to use `-insecure` to
-allow connecting to servers with self-signed or otherwise invalid TLS
-certificate. With `-tls`, you can also use the `-ca-bundle-path <path>`
-to use a specific CA bundle path. Adding the `-websocket` flag will
-cause NDT to wrap its messages (possibly already wrapped by JSON) into
-WebSocket messages. Finally, adding the `-ndt7` flag turns on version
-7 of the NDT protocol, which is not backwards compatible. Since `-ndt7`
-uses TLS, both `-ca-bundle-path <path>` and `-insecure` work also
-in combination with the `-ndt7` flag. When using `-ndt7`, `-batch` can be
-specified so that the only output on STDOUT will be the JSON test results.
-To further reduce the amount of output, you can use the `-summary` flag,
-which only prints a summary at the end of the tests. If used with `-batch`,
-the generated summary will be JSON.
+By default, we use the ndt7 protocol. Adding the `-tls` flag causes NDT to use
+a TLS connection rather than a TCP connection. When using `-tls`, you may also
+want to use `-insecure` to allow connecting to servers with self-signed or
+otherwise invalid TLS certificate. With `-tls`, you can also use the
+`-ca-bundle-path <path>` to use a specific CA bundle path. Adding the
+`-websocket` flag will cause NDT to wrap its messages (possibly already wrapped
+by JSON) into WebSocket messages. Finally, adding the `-ndt7` flag turns on
+version 7 of the NDT protocol, which is not backwards compatible. Since `-ndt7`
+uses TLS, both `-ca-bundle-path <path>` and `-insecure` work also in combination
+with the `-ndt7` flag. When using `-ndt7`, `-batch` can be specified so that the
+only output on STDOUT will be the JSON test results.  To further reduce the
+amount of output, you can use the `-summary` flag, which only prints a summary
+at the end of the tests. If used with `-batch`, the generated summary will be
+JSON.
 
 In practice, these are the flags you want to use:
 
 1. none, to use the original NDT protocol;
 
 2. `-tls` to use the original NDT protocol over TLS;
```

Comment:
This text seems unnecessary now

Reply:
This text will want significant updates soon. I've made it aspirational for now, since ultimately we want either "no flags" for ndt7 ws, or (for now) "-tls" for ndt7 wss.